Class/Practise 3.2/include/Triangle.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+class Triangle {
+public:
+    Triangle(double a, double b, double c);
+    double area() const;
```

Comment:
calculateArea may be a better name if we don't perform real calculations during construction - it shows that the function may take more time to execute not just instant return